apps/messaging/components/navigation/UserMenuDrawer.tsx

```diff
@@ -51,6 +54,18 @@ export default function UserMenuDrawer(
         asButton={{
           size: "large",
         }}
+        onClick={() => {
+          analyticsClient.trackEvent({
+            event: {
+              name: ANALYTICS[isPublicServant ? "adminUser" : "user"].logout
+                .name,
+              category:
+                ANALYTICS[isPublicServant ? "adminUser" : "user"].category,
+              action:
+                ANALYTICS[isPublicServant ? "adminUser" : "user"].logout.action,
+            },
+          })
+        }}
```

Comment:
What about something like this?

```suggestion
          const eventType = ANALYTICS[isPublicServant ? "adminUser" : "user"];
          analyticsClient.trackEvent({
            event: {
              name: eventType.logout.name,
              category: eventType.category,
              action: eventType.logout.action,
            },
          })
        }}
```

Reply:
fixed!